0_explore_data/cluster_by_accessory_gene.py

```diff
@@ -568,12 +568,12 @@
 )
 fig4 += pn.scales.scale_x_log10()
 fig4 += pn.scales.scale_y_log10()
-fig4 += pn.geom_point(pn.aes(color="Strain type_pao1"), alpha=0.4)
+fig4 += pn.geom_point(pn.aes(color="Strain type_pao1"), alpha=0.3, size=3, stroke=0.8)
```

Comment:
would it make sense to do an alpha of ~0.5? The points are super faint. Usually I would say dropping it is good because the points along the axes are so dense, but because these aren't the focus, and there isn't tons of overlap for the ones that don't overlap very much, maybe slightly higher color saturation could be nice? Feel free to ignore this if you want.